tests/lib/rules/no-render-in-setup.test.ts

```diff
@@ -0,0 +1,82 @@
+import { createRuleTester } from '../test-utils';
+import { BEFORE_HOOKS } from '../../../lib/utils';
+import rule, { RULE_NAME } from '../../../lib/rules/no-render-in-setup';
+
+const ruleTester = createRuleTester({
+  ecmaFeatures: {
+    jsx: true,
+  },
+});
+
+ruleTester.run(RULE_NAME, rule, {
+  valid: [
+    {
+      code: `
+        it('Test', () => {
+          render(<Component/>)
```

Comment:
Could you add a couple of additional valid test checking that the rule doesn't report a `render` used within `beforeEach` and `beforeAll` if that render method doesn't come from testing library? Something like

```javascript
import { render } from '@somewhere/else';

beforeEach(() => {
  render(<Component />);
});
```

This has been checked already in existing rules. Let me know if you need some help with this!